mirrord/agent/src/steal/connection.rs

```diff
@@ -86,6 +91,7 @@ impl TcpConnectionStealer {
             iptables: None, // Initialize on first subscription.
             write_streams: HashMap::with_capacity(8),
             read_streams: StreamMap::with_capacity(8),
+            connection_client: HashMap::with_capacity(8),
```

Comment:
Is this supposed to be `connection_clients`**?

Reply:
Should I change it to that? Called it `connection_client` because it gives you the single client of a connection, unlike `client_connections` which gives you the set of connections of a client.

Reply:
If it's a data structure containing many clients, each associates with a connection, then yeah, it should be `connection_clients` (and the other one should be `client_connectionses`, but alas).

Reply:
done